sklearn/impute/tests/test_impute.py

```diff
@@ -205,12 +209,12 @@ def test_imputation_median_special_cases():
     X = np.array([
         [0, np.nan, np.nan],  # odd: implicit zero
         [5, np.nan, np.nan],  # odd: explicit nonzero
-        [0, 0, np.nan],    # even: average two zeros
-        [-5, 0, np.nan],   # even: avg zero and neg
-        [0, 5, np.nan],    # even: avg zero and pos
-        [4, 5, np.nan],    # even: avg nonzeros
+        [0, 0, np.nan],  # even: average two zeros
```

Comment:
Please do not change unrelated things. It makes your contribution harder to review and may introduce merge conflicts to other pull requests.

Reply:
Not sure how that got in there but absolutely will revert that!